src/lib/data/createables.ts

```diff
@@ -2367,6 +2367,11 @@ const Createables: Createable[] = [
 			crafting: 35
 		}
 	},
+	{
+		name: 'Venator bow (uncharged)',
+		inputItems: new Bank().add('Venator shard', 5),
+		outputItems: new Bank().add('Venator bow (uncharged)')
```

Comment:
There's a mistake here.

Reply:
What's the mistake here? Do you mean you want it to create the charged version? The way it works currently is it created the uncharged version and then when you /minion charge it becomes the normal version.

Reply:
The only mistake I see here is that the Bank's don't end with .freeze(), however most of the creatables in that file don't have .freeze() at the end either.

eg. `inputItems: new Bank().add('Venator shard', 5).freeze(),`